client/src/i18n/index.js

```diff
@@ -7,6 +7,8 @@ import zhCN from './zh-CN';
 import nl from './nl';
 import ru from './ru';
 import esAR from './es-AR';
+import ja from './ja';
+
```

Comment:
Could you remove this line so the linter validates the file?

Reply:
OK, Sorry for the inconvenience.